vignettes/data_retrieval_and_utility_workflows.Rmd

```diff
@@ -167,5 +167,5 @@ metrics_gender <- get_subtype_metrics(
 )
 knitr::kable(metrics_gender)
 ```
-
+%% I was wondering if you can do continuous variables too, like average age? also could you get something like table(df$subtype, df$gender)?
```

Comment:
The intended behaviour of this function is to operate solely on categorical variables. This is justified in the function documentation, but perhaps this should also be clarified in this vignette example. Please add this.